customer.go

```diff
@@ -110,6 +111,20 @@ type Customer struct {
 	TaxInfoVerification *CustomerTaxInfoVerification `json:"tax_info_verification"`
 }
 
+// CustomerInvoiceCustomField represents a custom field associated with the customer's invoices.
+type CustomerInvoiceCustomField struct {
+	Name  *string `form:"name"`
+	Value *string `form:"value"`
+}
+
+// CustomerInvoiceSettings is the structure containing the default settings for invoices associated
+// with this customer.
+type CustomerInvoiceSettings struct {
+	CustomFields         []*CustomerInvoiceCustomField `json:"custom_fields"`
+	DefaultPaymentMethod *PaymentMethod                `json:"default_payment_method"`
+	Footer               string                        `json:"footer"`
+}
+
```

Comment:
We miss expandable `default_payment_method` on Customer struct here.
Or do you leave it out intentionally?

Reply:
Oh whoops my bad. It is correct here!